querybook/server/models/event_log.py

```diff
@@ -0,0 +1,26 @@
+import sqlalchemy as sql
+
+from app import db
+
+from const.db import now
+from const.event_log import EventType
+
+Base = db.Base
+
+
+class EventLog(Base):
+    __tablename__ = "event_log"
+    id = sql.Column(sql.Integer, primary_key=True)
+    created_at = sql.Column(sql.DateTime, default=now)
+    uid = sql.Column(sql.Integer, sql.ForeignKey("user.id"))
```

Comment:
on delete cascade

Reply:
i think the default is on delete fail, you need to change it to on delete set null?

Reply:
If we are going to keep them anyway, maybe just remove the foreign key constraint